client/src/components/Cardify.js

```diff
@@ -1,14 +1,14 @@
-import React from 'react';
-import { Card } from 'reactstrap';
+// import React from 'react';
+// import { Card } from 'reactstrap';
 
-const Cardify = (props) => {
-  return (
-    <div>
-      <Card>
-        {props.children}
-      </Card> 
-    </div>
-   );
-}
+// const Cardify = (props) => {
+//   return (
+//     <div>
+//       <Card>
+//         {props.children}
+//       </Card> 
+//     </div>
+//    );
+// }
```

Comment:
Let's delete this file, use git rm <filename> instead of just deleting it to make sure git removes it

Reply:
Done